sp/src/game/server/episodic/npc_hunter.cpp

```diff
@@ -68,6 +68,7 @@
 #endif
 #ifdef EZ2
 #include "npc_playercompanion.h"
+#include "ez2/npc_husk_base.h"
```

Comment:
I don't think this file has been merged yet

Reply:
Good call